ansible/playbooks/aix.yml

```diff
@@ -440,29 +440,33 @@
       # x11.adt.ext installp download requiring an IBMid                      #
       # https://www-01.ibm.com/support/docview.wss?uid=isg1fileset-1198156818 #
       #########################################################################
```

Comment:
Can you update the comment to detail where this content was pulled from?  How the OpenGL_X11.tar file was created.  Please work with @sxa555 for getting this file updated on the Adopt AWX server.

Reply:
Updated with where content was pulled from

Reply:
I don't see anything updated in the comment.  Guessing this is what could be used?
`http://www-01.ibm.com/support/docview.wss?uid=isg1fileset-2054486724`